test/core/base/check.js

```diff
@@ -274,6 +274,124 @@ describe('Check', function() {
 				});
 			});
 		});
+
+		describe('runSync', function() {
```

Comment:
You're not testing the `virtualNode`

Reply:
These test are copy pasted from the async version of the test. Does that mean the async version is not testing `virtualNode`?

Reply:
Jup, sounds like it. What I think is missing is just a test that the virtualNode is passed to the evaluate function correctly.